tests/a11y/cypress/support/e2e.js

```diff
@@ -19,3 +19,7 @@ import "./commands";
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+// Ignore uncaught script errors. We can't control what happens with external
+// scripts, so...
+Cypress.on("uncaught:exception", () => false);
```

Comment:
This makes me sad, but for now, we get an unhandled script error from DAP which causes the Cypress tests to fail.